api/python/tests/test_indigo.py

```diff
@@ -9,3 +9,7 @@ def test_aromatize_smiles(self) -> None:
         m = self.indigo.loadMolecule('C1=CC=CC=C1')
         m.aromatize()
         self.assertEqual('c1ccccc1', m.smiles())
+
+    def test_check_salt(self) -> None:
+        m = self.indigo.loadMolecule('[Na+].C')
+        self.assertTrue(m.checkSalt())
```

Comment:
Let's add a couple more tests:
1. To return `True` if compound contains multiple salts.
2. To return `False` if compound does not contain any salts.
3. To return `False` if compound contains, some salt-like ion, but not as standalone fragment.

Reply:
I'll add them, thank you!